recipes/vulkan-validationlayers/all/conanfile.py

```diff
@@ -0,0 +1,130 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import glob
+import os
+import shutil
+
+
+class VulkanValidationLayersConan(ConanFile):
+    name = "vulkan-validationlayers"
+    description = "Khronos official Vulkan validation layers for Windows, Linux, Android, and MacOS."
+    license = "Apache-2.0"
+    topics = ("conan", "vulkan-validation-layers", "vulkan", "validation-layers")
+    homepage = "https://github.com/KhronosGroup/Vulkan-ValidationLayers"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "with_wsi_xcb": [True, False],
+        "with_wsi_xlib": [True, False],
+        "with_wsi_wayland": [True, False]
+    }
+    default_options = {
+        "with_wsi_xcb": True,
+        "with_wsi_xlib": True,
+        "with_wsi_wayland": True
+    }
+
+    short_paths = True
+
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os != "Linux":
+            del self.options.with_wsi_xcb
+            del self.options.with_wsi_xlib
+            del self.options.with_wsi_wayland
+
+    def configure(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 11)
+        if self.settings.compiler == "gcc" and tools.Version(self.settings.compiler.version) < "5":
+            raise ConanInvalidConfiguration("gcc < 5 is not supported")
+
+    def requirements(self):
+        self.requires("glslang/8.13.3559")
```

Comment:
Actually, it seems that glslang is not required (only for tests). I'll try some tricks to make upstream CMakeLists happy with a fake glslang interface target.

Reply:
Moreover, spirv-headers is also a direct dependency of vulkan-validationlayers, but:
- spirv-tools also depends on spirv-headers
- spirv-tools is very strict on spirv-headers version, so I would prefer to not introduce conflicts.